django/applications/catmaid/admin.py

```diff
@@ -51,10 +52,10 @@ def duplicate_action(modeladmin, request, queryset):
     for object in queryset:
         object.id = None
         object.save()
-duplicate_action.short_description = "Duplicate selected without relations"
+duplicate_action.short_description = "Duplicate selected without relations" # type: ignore # https://github.com/python/mypy/issues/2087
 
 
-def export_project_json_action(modeladmin, requet, queryset):
+def export_project_json_action(modeladmin, request, queryset) -> HttpResponse:
```

Comment:
Also, this parameter is unused

Reply:
I think this interface is asked for by Django.